dbms/src/Storages/DeltaMerge/SegmentReadTaskPool.h

```diff
@@ -111,6 +111,13 @@ class BlockStat
     std::atomic<int64_t> total_bytes;
 };
 
+enum class ReadMode
+{
+    Normal, // Read in Normal Mode.
+    Fast, // Read in Fast Mode, read data without mvcc and sorted merge, but filter the data whose del_mark = 1.
+    Raw, // Just for 'selraw * ', always using in debug code.
```

Comment:
By using `///` or `/** */` pattern this could be a doc comment instead of just a comment. Many common editors and IDEs adopted this style, see https://doxygen.nl/manual/docblocks.html

```suggestion
    /**
     * Read in normal mode. Data is ordered by PK, and only the most recent version is returned.
     */
    Normal,

    /**
     * Read in fast mode. Data is not sort merged, and all versions are returned. However, deleted records (del_mark=1)
     * will be still filtered out.
     */
    Fast,
    
    /**
     * Read in raw mode, for example, for statements like `SELRAW *`. In raw mode, data is not sort merged and all versions
     * are just returned.
     */
    Raw,
```